modules/tfe-instance/aws.tf

```diff
@@ -277,13 +270,16 @@ KMS_KEY_ID="${var.kms_key_id}"
 INSTALL_ID="${var.installation_id}"
 DATA_REDUNDANCY="${var.ebs_redundancy}"
 PROXY_URL="${var.proxy_url}"
-    BASH
+  BASH
 }
 
 resource "aws_elb" "ptfe" {
   internal        = "${var.internal_elb}"
   subnets         = ["${var.elb_subnet_id}"]
-  security_groups = ["${coalesce(var.external_security_group_id, join("", aws_security_group.ptfe-external.*.id))}"]
+  security_groups = [
```

Comment:
Is the compact list flow above better for this? I was unsure which is preferred.